tests/test_plugins.py

```diff
@@ -21,6 +21,7 @@
     straxen.aux_repo + 'f0df03e1f45b5bdd9be364c5caefdaf3c74e044e/fax_files/mlp_model.json',
     nn_weights=
     straxen.aux_repo + 'f0df03e1f45b5bdd9be364c5caefdaf3c74e044e/fax_files/mlp_model.h5',
+    nn_file=None,

```

Comment:
This was failing the tests as we don't have access to this NN on our public repo (as desired)

Reply:
Thanks for fixing it!